src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,225 @@
+
+Badges tend to be more user-friendly than notifications, and can be updated
+with a much higher frequency, since they don't interrupt the user. And,
+because they don't interrupt the user, they don't need the user's permission.
+
+### Possible use cases {: #use-cases }
+
+Examples of sites that may use this API include:
+
+* Chat, email, and social apps, to signal that new messages have arrived, or to
+  show the number of unread items.
+* Productivity apps, to signal that a long-running background task (such as
+  rendering an image or video) has completed.
+* Games, to signal that a player action is required (e.g., in Chess, when it
+  is the player's turn).
+
+## Current status {: #status }
+
+| Step                                       | Status                       |
+| ------------------------------------------ | ---------------------------- |
+| 1. Create explainer                        | [Complete][explainer]        |
+| 2. Create initial draft of specification   | [Complete][spec]             |
+| 3. Gather feedback & iterate on design     | [In progress](#feedback)     |
+| **4. Origin trial**                        | [**In progress**](#ot)       |
+| 5. Launch                                  | Not started                  |
+
+{% Aside 'codelab' %}
+See it in action
+{% endAside %}
+
+1. Using Chrome 73 or later on Windows or Mac, open the [Badging API demo][demo].
+2. When prompted, click **Install** to install the app, or use the Chrome
+   menu to install it.
+3. Open it as an installed PWA. Note, it must be running as an installed PWA (in
+   your task bar or dock).
+4. Click the **Set** or **Clear** button to set or clear the badge from the app
+   icon. You can also provide a number for the *Badge value*.
+
+{% Aside %}
+  While the Badging API *in Chrome* requires an installed app
+  with an icon that can actually be badged, you shouldn't
+  make calls to the Badging API dependent on the install state.
+  The Badging API can apply to *anywhere* a browser might want to show a badge,
+  so developers shouldn't make any assumptions about situations where
+  the browser will display them. Just call the API when it exists.
+  If it works, it works. If not, it simply doesn't.
+{% endAside %}
+
+
+## How to use the Badging API {: #use }
+
+Starting in Chrome 73, the Badging API is available as an origin trial
+for Windows (7+) and macOS.
+
+[Origin trials][ot-what-is] allow you to try out new features and give
+feedback on usability, practicality, and effectiveness to the Chrome team and the web
+standards community. For more information, see the
+[Origin Trials Guide for Web Developers][ot-dev-guide].
+
+{% Aside 'caution' %}
+  Android is not supported because it requires you to show a notification,
+  though this may change in the future. Chrome OS support is pending
+  implementation of badging on the platform.
+{% endAside %}
+
+### Register for the origin trial {: #ot }
+
+1. [Request a token][ot-request] for your origin.
```

Comment:
Will all of the capabilities posts have a block like this? If so we might want to also put it into an include.

Reply:
@robdodson see #1618

Reply:
Putting it into an include would be nice, as long as we can set a variable or something on the include so that it links directly to the origin trial we're referencing.